site/gatsby-site/src/components/sidebar/index.js

```diff
@@ -9,8 +9,65 @@ import { faChevronLeft } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { useUserContext } from 'contexts/userContext';
 import { useMenuContext } from 'contexts/MenuContext';
+import { graphql, useStaticQuery } from 'gatsby';
 
 const Sidebar = ({ defaultCollapsed = false, location = null, setNavCollapsed }) => {
+  const navConfig = config.sidebar.navConfig;
+
+  const { sidebar } = useStaticQuery(graphql`
+    {
+      sidebar: allPrismicSidebar(sort: { data: { order: ASC } }) {
+        edges {
+          node {
+            data {
+              title
+              label
+              url {
+                url
+              }
+              path
+              order
+              items {
+                item_title
+                item_label
+                item_url {
+                  url
+                }
+                item_path
+              }
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  let sidebarItems = [];
+
+  if (sidebar.edges.length > 0) {
+    sidebarItems = sidebar.edges.map((item) => {
+      if (item.node?.data) {
```

Comment:
Is there an instance where this is not present? if this happens, `map` will add an `undefined` item.

Reply:
No, it shouldn't happen because this is part of the prismic structure so if there's an item, it has node.data
I removed the if